NerlnetInstall.sh

```diff
@@ -68,4 +68,4 @@ echo "enable and start nerlnet.service"
 systemctl enable nerlnet.service
 #systemctl start nerlnet.service
 
-rm *.deb
```

Comment:
@dolby360 

The new NerlnetInstall.sh installs erlang from source only by demand using --i or --install.
Therefore, no need to remove deb files. 
This branch should be rebased on top of master since there are new commits (with the new NerlnetInstall.sh) merged Yesterday. 
I think it's better to:
- revert this change.
- rebase on master.
- inspect the updated NerlnetInstall.sh script.